internal/driver/glfw/window.go

```diff
@@ -160,13 +160,13 @@ func (w *window) RequestFocus() {
 func (w *window) Resize(size fyne.Size) {
 	w.canvas.Resize(size)
 	w.viewLock.Lock()
-	w.width, w.height = internal.ScaleInt(w.canvas, size.Width), internal.ScaleInt(w.canvas, size.Height)
+	if w.fixedSize {
```

Comment:
If fixedSize, change window size. This seems wrong, but maybe I'm misunderstanding what fixedSize means?

Reply:
fixedSize will ignore "user generated" window sizes, so here we save the "correct size" as future events may be incorrect.

I.e. set a fixed size - then the WM may show window small before getting our expand signal - so we have to save the state explicitly.

Reply:
okay so fixedSize means not user resizable, but it can still be resized programmatically?

Reply:
Yes. it has to be, otherwise you are forcing fixed size at minsize.
You would do (for example) w.Resize(); w.SetFixedSize(true) (but in either order).